ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala

```diff
@@ -10,7 +10,7 @@ import com.daml.metrics.api.{MetricDoc, MetricHandle, MetricName}
 class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: MetricsFactory) {
 
   @MetricDoc.Tag(
-    summary = "Number of events that will be metered",
+    summary = "Number of ledger events that are metered",
```

Comment:
For consistency (we seem to enter full sentences as `summary`):
```suggestion
    summary = "Number of ledger events that are metered.",
```